metadata/2023-06-05-reenable-signature-checker-v2/reenable_sig_checker_v2.json

```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Re-enable the new signature checker",
+  "description": "The new signature checker was disabled by a previous proposal due to some backward compatibility issue. That issue has now been resolved, so it is time to re-enable the new signature checker.",
```

Comment:
Can you add a couple sentences to explain why we want the new signature checker?

Reply:
Done